Parse/PFObject.h

```diff
@@ -813,6 +813,18 @@ NS_REQUIRES_PROPERTY_DEFINITIONS
  */
 - (BFTask *)deleteEventually;
 
+/*!
```

Comment:
Can we move these to straight after `objectForKey:`, `setObjectForKey:`, as they are not really related to `**Eventually`.